Editor/AssetBundleModel/ABModelAssetInfo.cs

```diff
@@ -176,7 +177,8 @@ internal IEnumerable<MessageSystem.Message> GetMessages()
             if (m_dependencies != null && m_dependencies.Count > 0)
             {
                 var message = string.Empty;
-                foreach (var dependent in m_dependencies)
+                List<AssetInfo> sortedDependencies = m_dependencies.OrderBy(d => d.bundleName).ToList();
```

Comment:
The .ToList() is redundant, since the foreach will work with the default return value of `OrderBy`. We also tend to use `var` for local variables - so this could be simplified just a bit to `var sortedDependencies = m_dependencies.OrderBy(d => d.bundleName);`

Reply:
Thanks, those are good points. I've updated them to keep it consistent with the rest!